source/developers/concepts/hooks_extension_framework.rst

```diff
@@ -163,6 +163,38 @@ Filters are useful when you need an immediate response that directly modifies th
 
 We encourage you to review the list of use cases for events and filters to draw inspiration from real-life scenarios and see if your use case aligns with any of them. Also, maybe your feature can be implementing using the framework but there's not an available event or filter for it yet. In that case, consider proposing a new event or filter to the community!
 
+What's Next?
+============
+
+Contribute to the Hooks Extension Framework
+-------------------------------------------
+
+Before considering contributing to the framework, we suggest you make a quick proposal with your use case and intended solution, and share it with the community to gather feedback and validate the approach. This can be done in the Open edX Slack channels, the Open edX Discuss forum or in each repository's issue tracker.
+
+If you're new to contributing to the Hooks Extension Framework, we recommend also reading the :doc:`openedx-events:index` and :doc:`openedx-filters:index` documentation to understand each hook concepts and how to use them in your project.
```

Comment:
In some places, the default link text I think is not the best, for example:
![image](https://github.com/user-attachments/assets/806683cc-a0a0-495a-a76d-f846523f2186)

This would look better:
![image](https://github.com/user-attachments/assets/5cc2da3f-9d4f-4f97-89d1-d90f21b0a810)